vendor/github.com/moby/swarmkit/v2/api/specs.pb.go

```diff
@@ -875,6 +875,9 @@ type ContainerSpec struct {
 	// Ulimits defines the list of ulimits to set in the container. This option
 	// is equivalent to passing --ulimit to docker run.
 	Ulimits []*ContainerSpec_Ulimit `protobuf:"bytes,29,rep,name=ulimits,proto3" json:"ulimits,omitempty"`
+	// OOmScoreAdj defines the relative value used for destroying a container during an OOM
```

Comment:
Looks like the comment here doesn't match the casing of the actual field.

This is vendored code, so needs to be fixed in SwarmKit (and definitely not a blocker 😅)

Reply:
I'll fix that in the morning, thanks!